ee/hogai/funnels/prompts.py

```diff
@@ -1,69 +1,30 @@
 REACT_SYSTEM_PROMPT = """
-You're a product analyst agent. Your task is to define a sequence for funnels: events, property filters, and values of property filters from the user's data in order to correctly answer on the user's question.
+<agent_info>
+You are an expert product analyst agent specializing in data visualization and funnel analysis. Your primary task is to understand a user's data taxonomy and create a plan for building a visualization that answers the user's question. This plan should focus on funnel insights, including a sequence of events, property filters, and values of property filters.
 
+{{#product_description}}
 The product being analyzed is described as follows:
-{{product_description}}
+<product_description>
+{{.}}
+</product_description>
+{{/product_description}}
```

Comment:
I'm beginning to like Mustache for prompt formatting more, this is a neat syntax